sql/mssql/00020-Evaluation-Tables.sql

```diff
@@ -0,0 +1,149 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE eppeta.PerformanceEvaluation (
+    EducationOrganizationId INT NOT NULL,
+    EvaluationPeriodDescriptorId INT NOT NULL,
+    PerformanceEvaluationTitle VARCHAR(50) NOT NULL,
+    PerformanceEvaluationTypeDescriptorId INT NOT NULL,
+    SchoolYear SMALLINT NOT NULL,
+    TermDescriptorId INT NOT NULL,
+    PerformanceEvaluationDescription VARCHAR(255) NULL,
+    CreateDate DATETIME2 NOT NULL,
+    LastModifiedDate DATETIME2 NOT NULL,
+    Ods_Id UNIQUEIDENTIFIER NULL,
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+);
+GO
+ALTER TABLE eppeta.PerformanceEvaluation ADD CONSTRAINT [PerformanceEvaluation_DF_CreateDate] DEFAULT (getdate()) FOR [CreateDate]
+GO
+ALTER TABLE eppeta.PerformanceEvaluation ADD CONSTRAINT [PerformanceEvaluation_DF_LastModifiedDate] DEFAULT (getdate()) FOR [LastModifiedDate]
+GO
+
+CREATE TABLE eppeta.PerformanceEvaluationRating (
+    EducationOrganizationId INT NOT NULL,
+    EvaluationPeriodDescriptorId INT NOT NULL,
+    PerformanceEvaluationTitle VARCHAR(50) NOT NULL,
+    PerformanceEvaluationTypeDescriptorId INT NOT NULL,
+    SchoolYear SMALLINT NOT NULL,
+    TermDescriptorId INT NOT NULL,
+    ActualDate DATE NOT NULL,
+    ActualDuration INT NULL,
+    PerformanceEvaluationRatingLevelDescriptorId INT NULL,
+    ActualTime TIME NULL,
```

Comment:
I have been using SQL Server continuously since 2005 and I did not know there is a `TIME` data type. I have literally never seen it be used before 😄. It might be better to just leave this as a string or a numeric type, depending on how you are going to use it in C#.

Reply:
![image](https://github.com/Ed-Fi-Exchange-OSS/EvaluationTracker/assets/48261255/05555632-0762-4afb-a404-0985ee29832d)
This will be a user entered field that might be used to calculate duration within this app and then transferred through a string field on the API and will land in a TIME field within the ODS. I took the ODS SQL and trimmed it down to get these tables. I have no real opinion on TIME vs another datatype here

Reply:
If the value is already a `TIME` data type in the ODS then sounds fine to me to keep it that way.